style/header.css

```diff
@@ -86,7 +86,7 @@
   border-color: var(--primary-color-brand-01);
 }
 
-#burger-menu-check {
+.hidden-input {
```

Comment:
this should be a general class, not belong to the header

Reply:
Done